src/styles/blocks/card.scss

```diff
@@ -0,0 +1,72 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  padding: 16px;
+  width: 200px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin-bottom: 46px;
+  margin-right: 48px;
+  &_product-name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: #060b35;
+    margin-bottom: 4px;
+    margin-block-start: 0;
+  }
+  &_product-code {
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: #616070;
+  }
+  &_product-image {
+    width: 160px;
+    height: 134px;
+    margin: 16px auto 40px;
+  }
+  &_product_price {
+    display: flex;
+    justify-content: space-between;
+  }
+  &_product-buy-button {
+    display: flex;
+    background-color: #00acdc;
+    color: white;
+    text-transform: uppercase;
+    text-decoration: none;
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: 700;
+    border: 1px solid transparent;
+    border-radius: 5px;
+    width: 166px;
+    height: 40px;
+    justify-content: center;
+    align-items: center;
+    margin-top: 16px;
+  }
+  &_product-buy-button:hover {
+    background-color: white;
+    color: #00acdc;
+    border-color: #00acdc;
+  }
+}
+
+.card:nth-child(4n) {
+  margin-right: 0;
+}
+
+.price {
```

Comment:
[checklist p9](https://github.com/mate-academy/layout_catalog/blob/master/checklist.md)

@Anna-Tkachenko has wroten twice that each BEM block should have separate files. Be more attentive to comments, if something is not clear for u fill free to ask in slack or visit Q&A